package/health_daemon/src/Makefile

```diff
@@ -1,5 +1,5 @@
 TARGET=health_daemon
-SOURCES=health_daemon.c health_monitor.c baseline_monitor.c
+SOURCES=health_daemon.c health_monitor.c baseline_monitor.c glo_bias_monitor.c glo_obs_monitor.c utils.c
 LIBS=-lczmq -lsbp -lpiksi -lm
 CFLAGS=-std=gnu11 -Wmissing-prototypes -Wimplicit -Wshadow -Wswitch-default -Wswitch-enum -Wundef -Wuninitialized -Wpointer-arith -Wstrict-prototypes -Wcast-align -Wformat=2 -Wimplicit-function-declaration -Wredundant-decls -Wformat-security -Wall -Wextra -Wno-strict-prototypes -Werror
```

Comment:
Can we add `-Wconversion`?  This would catch the issue mentioned by @anth-cole, I believe.

Reply:
Sorry guys, I was trying to get this in for testing before HITL gets torn down tonight. Merged before I saw these comments. Will add a quick PR for review.

Reply:
Ironically this cropped up a bunch of errors, but sadly it did not catch the implicit cast mentioned above. Anyways, it is in there now, and merged.

Reply:
👍